pages/mesosphere/dcos/services/edge-lb/1.5/concepts/self-service/index.md

```diff
@@ -0,0 +1,327 @@
+---
+layout: layout.pug
+navigationTitle: Self Service Auto Pools
+title: Self Service Auto Pools
+menuWeight: 1
+excerpt: Describes how you can automaticly expose applications with task labels
+enterprise: true
+---
+
+Edge-LB supports automatic provisioning of the pools based on the Marathon app labels / SDK tasks labels (annotations). Pool configuration is driven from a pool template. A default pool template is created at the time of Edge-LB installation. The template is rendered into a pool configuration based on the values of the app / task [labels](#supported-labels) (discussed below).
```

Comment:
I'd link `pools` to `architecture/#edge-lb-pool` for those less fluent with the concepts.